### ISSUE TITLE
Don't show the add manga pop up when long pressing a manga already in database

### ISSUE THREAD

**Issue by Dostoi** [raulhaag/MiMangaNu#361]
In manga list in the server result, if you long press a manga, you have a pop-up asking you if you want to add it in your database.

It would be more logical to avoid showing the add manga pop-up if it's already in the database but instead show a message saying so.

**Comment by xtj-9182** [comment]
I would really like to implement this but I don't think it's possible.
I've thought about this before and I did some testing. I just retested it again and I came to the same conclusion; I think it's not possible. When you long press on a manga in the server list, the first thing that happens is that a thread is started to find out if the manga is already added or not, immediately after that we construct the context menu with the add text. I've did some testing and it seems that the context menu is constructed too fast to know if the manga is added or not, the thread isn't done with it's processing when the menu is already constructed and from what I can tell we can't change the value of the add text after the context menu is constructed. If we could know if the manga is added before we need to construct the menu then we could exchange it with a manga is already added text but alas that seems unfeasible. The reason why this works for the DetailsFragment is that it usually takes a few moments before it finishes loading which is most often enough time for the thread to finish processing.
